manifests/component/lookup_table.pp

```diff
@@ -5,13 +5,21 @@
   Hash             $lookup_json,
   String           $lookup_file,
   Boolean          $reload_on_hup,
+  Boolean          $rsyslog_in_docker = false,
+  Optional[String] $json_file = undef,
```

Comment:
why not `Optional[Stdlib::AbsolutePath]` ?

Reply:
fixed